server/game/cards/04.3-TFW/Smoke.js

```diff
@@ -1,10 +1,21 @@
 const DrawCard = require('../../drawcard.js');
 
 class Smoke extends DrawCard {
-    setupCardAbilities(ability) { // eslint-disable-line no-unused-vars
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Give non-unique characters -2/+0',
+            condition: context => this.game.isDuringConflict() && context.source.parent && context.source.parent.isParticipating(),
+            cost: [ability.costs.bowSelf(), ability.costs.sacrificeSelf()],
+            gameAction: ability.actions.cardLastingEffect(context => ({
+                target: context.game.currentConflict.getParticipants().filter(card => !card.isUnique()),
+                effect: ability.effects.modifyMilitarySkill(-2)
+            })),
+            effect: 'give all non-unique participating characters -2{1}',
```

Comment:
What's the reasoning for passing in the word 'military'? Why not just write it?

Reply:
Passing in military converts it to the symbol, if I just wrote 'military' in effect I believe it remains as the word 'military'. Asahina Artisan and Captive Audience do this as well.

Reply:
Ah, okay.

Reply:
That's correct. I'm not particularly happy with the way that works, but I didn't write that code and I'd already made a lot of changes...